types/kue/index.d.ts

```diff
@@ -78,6 +78,8 @@ export declare class Job extends events.EventEmitter {
     public failed_at: string | number;
     public started_at: string | number;
     public client: redisClientFactory.RedisClient;
+    public _error: string;
```

Comment:
Properties starting with `_` are meant to be private.
Please change it to `private`.